documentation/CAMARA-OIDC-profile.md

```diff
@@ -0,0 +1,1022 @@
+
+* **grant_type**
+
+  REQUIRED. Value MUST be set to `authorization_code`.
+
+
+* **code**
+
+  REQUIRED. The authorization code received from the authorization server.
+
+
+* **redirect_uri**
+
+  REQUIRED, if the `redirect_uri` parameter was included in the authorization request, and their values MUST be identical.
+
+
+The client MUST authenticate with the authorization server as described in the [Client Authentication Section](#client-authentication).
+
+For example, the client makes the following HTTP request (with line wraps within values for display purposes only):
+
+```http
+POST /token HTTP/1.1 
+Host: server.example.com 
+Content-Type: application/x-www-form-urlencoded 
+
+grant_type=authorization_code&code=SplxlOBeZQQYbYS6WxSbIA&
+redirect_uri=https%3A%2F%2Fclient%2Eexample%2Ecom%2Fcb&
+client_assertion_type=urn%3Aietf%3Aparams%3Aoauth%3Aclient-assertion-type%3Ajwt-bearer&
+client_assertion=eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzNkJoZFJrcXQzIiwiYXVkIjoiaHR0cHM6Ly9zZXJ2ZXIuZXhhbXBsZS5jb20iLCJzdWIiOiJzNkJoZFJrcXQzIiwiZXhwIjoxMzAwODE5MzgwLCJpYXQiOjEzMDA4MTkwODAsImp0aSI6ImUyMjA5MTNmLTM5ZjItNDEwNi05ZmU0LWI3YmEzNGJkZjJkNSJ9.khzDX5keZQ6EJiJd8upzUMvAZtH8-PypGrWJYeA-Upx1MRDEr54U0V3T2-XZ93PbQE0waMyn25Z-xiDsCMVazYvOEa_Hz5_mE2x1aVZ8m86Ze4lcLld91w0PwnTKqiYy5-vmauruNJCOFXugO1vc3qytA8m8631LhcbrDBHLLS5ieeeA0dXxDTacTPATb1TfM45ahj3Sb0DBA8pefMxuDRkrUAtMporaN4zJ-EenrB0TGlMXklF-w0XQnXezB4_sZmSKlopH4pfSMBKjj_fScP1OlJydKkrbLQzMNHzY6IIgN_xv2Ed-Rf4_I9jdGC8r3BV4YLlQYklKaFfObr86Ew 
+```
+
+### Access Token Response
+
+#### Successful Response
+
+The authorization server issues an access token and optional refresh token and ID Token, and constructs the response by adding the following parameters to the entity-body of the HTTP response with a 200 (OK) status code:
+
+* **access_token**
+
+  REQUIRED. The access token issued by the authorization server.
+
+
+* **token_type**
+
+  REQUIRED. The type of the token issued. It MUST be `bearer`.
+
+
+* **expires_in**
+
+  RECOMMENDED. The lifetime in seconds of the access token.
+
+
+* **refresh_token**
+
+  OPTIONAL, if scope `offline_access` is not included; otherwise, REQUIRED. The refresh token, which can be used to obtain new access tokens using the same authorization.
+
+
+* **scope**
+
+  OPTIONAL, if identical to the scope requested by the client; otherwise, REQUIRED.
+
+
+* **id_token**
+
+  OPTIONAL, if scope `openid` is not included; otherwise, REQUIRED. ID Token value associated with the authenticated session.
+
+
+The following is a non-normative example of a successful Token Response (the ID Token signature in the example can be verified with the key at [Appendix A.1 Authorization Server RSA Key](#appendix-a1-authorization-server-rsa-key)).
+
+```http 
+HTTP/1.1 200 OK
+Content-Type: application/json
+```
+```json
+{
+  "access_token": "SlAV32hkKG",
+  "token_type": "Bearer",
+  "refresh_token": "8xLOxBtZp8",
+  "expires_in": 3600,
```

Comment:
The 10-minute recommendation only applies to the authorization code. If you want to raise the debate of token lifetime recommendations, it may be best to open a new issue to discuss it so that it can be included in future versions of the profile.